domain-models-runtime/src/main/resources/templates/db_scripts/optimistic_locking.ftl

```diff
@@ -39,7 +39,7 @@
   DROP TRIGGER IF EXISTS set_${table.tableName}_ol_version_trigger
     ON ${myuniversity}_${mymodule}.${table.tableName} CASCADE;
     
-  CREATE TRIGGER set_${table.tableName}_ol_version_trigger BEFORE INSERT OR UPDATE
+  CREATE TRIGGER set_${table.tableName}_ol_version_trigger BEFORE INSERT OR UPDATE OF jsonb
```

Comment:
This seems like a pretty significant change if this disables OL for any field that isn't a jsonb field. What am I not understanding? What does this mean for modules that are relying on OL for non-jsonb fields?

Reply:
Yes thanks. Just in case others come back to this PR and had similar questions: the existing API only contains derived columns (columns that are derived from other columns) and jsonb. Reducing the scope of _version to jsonb doesn’t affect existing code since these columns didn’t need OL in the first place, and it helps with changes that need to be made to mod-inventory.